### PR TITLE
fix: default configs emitter and definition path

### PR DESCRIPTION
**Pull request by gsanchezgavier**
Adds default value for:
* DefinitionFilesPath
* Emitter: uses `infra-sdk` when standalone is false and `telemetry` when is true (backward compatibility)

This PR fixes issue #83

### CODE REVIEW

**Review by paologallinaharbur — DISMISSED** [review]
LGTM!

File: internal/cmd/scraper/scraper.go [hunk 1]
```diff
@@ -296,7 +296,7 @@ func Run(cfg *Config) error {
 		case "infra-sdk":
 			specs, err := integration.LoadSpecFiles(cfg.DefinitionFilesPath)
 			if err != nil {
-				return err
+				logrus.Errorf("error loading definition files: %s", err)
```
Comment (paologallinaharbur) [inline internal/cmd/scraper/scraper.go:299]: Do we want to carry on even if the file is not found/not loaded?

Comment (gsanchezgavier) [reply internal/cmd/scraper/scraper.go:299]: i don't see why not. When the scraped target has no matching on the definitions the metrics are added to the host entity. Also this is checking that the folder is there

Comment (paologallinaharbur) [reply internal/cmd/scraper/scraper.go:299]: Yes that is right 👍

**Review by paologallinaharbur — APPROVED** [review]
thanks for the changes

**Review by ardias — APPROVED** [review]

**Pull request merged**